### PR TITLE
fix: Chinese traditional shows multiple durations of the event as a string

### LINKED ISSUE #16184: [CAL-4138] Cal.com in Chinese traditional shows multiple durations of the event as a string
<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-4138](https://linear.app/calcom/issue/CAL-4138/calcom-in-chinese-traditional-shows-multiple-durations-of-the-event-as)</sub>

### PR DESCRIPTION
**Pull request by zerone0x**
## What does this PR do?

- Fixes [[CAL-4138] Cal.com in Chinese traditional shows multiple durations of the event as a string #16184](https://github.com/calcom/cal.com/issues/16184)
- Fixes CAL-4138 (Linear issue number - should be visible at the bottom of the GitHub issue description)

## Mandatory Tasks (DO NOT REMOVE)

- [x] I have self-reviewed the code (A decent size PR without self-review might be rejected).
- [x] I have added a Docs issue [here](https://github.com/calcom/docs/issues/new) if this PR makes changes that would require a [documentation change](https://docs.cal.com). If N/A, write N/A here and check the checkbox.
- [x] I confirm automated tests are in place that prove my fix is effective or that my feature works.

## How should this be tested?

The local json of Chinese lack of map of `multiple_duration_timeUnit`. I added it to fix this i8n bug

### CODE REVIEW

**Review by Amit91848 — DISMISSED** [review]
LGTM! Thanks @zerone0x

**Review by anikdhabal — CHANGES_REQUESTED** [review]
Hey @zerone0x, thanks for your work! This text is very crucial and currently misses other locales as well. Please add the other locales that we support. It's very easy; you just need to copy and paste the same text.

**Comment by zerone0x** [comment]
Hi @anikdhabal I did a quick check for it, and I found: not only this one, but also `multiple_duration_mins`, `multiple_duration_timeUnit` missed for 32 times which I have added for 2 times, and `multiple_duration_mins` missed for 13 times.

**Comment by zerone0x** [comment]
Maybe a script need to be used to add it.🤔

**Comment by zerone0x** [comment]
Use a script to scan and complete all Json files, pls check

File: apps/web/public/static/locales/tr/common.json [hunk 1]
```diff
@@ -2099,5 +2099,6 @@
   "email_team_invite|content|invited_to_regular_team": "{{invitedBy}}, sizi {{appName}} uygulamasındaki `{{teamName}}` ekibine katılmaya davet etti. {{appName}}, size ve ekibinize e-posta iletişimine ihtiyaç duymadan toplantı planlama yapma olanağı sağlayan bir etkinlik planlayıcıdır.",
   "privacy": "Gizlilik",
   "signing_up_terms": "Devam ederek <0>Şartlar</0> ve <1>Gizlilik Politikası<1>'nı kabul etmiş olursunuz",
-  "ADD_NEW_STRINGS_ABOVE_THIS_LINE_TO_PREVENT_MERGE_CONFLICTS": "↑↑↑↑↑↑↑↑↑↑↑↑↑ Yeni dizelerinizi yukarıya ekleyin ↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑"
-}
+  "ADD_NEW_STRINGS_ABOVE_THIS_LINE_TO_PREVENT_MERGE_CONFLICTS": "↑↑↑↑↑↑↑↑↑↑↑↑↑ Yeni dizelerinizi yukarıya ekleyin ↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑",
+  "multiple_duration_timeUnit": "{{count}} $t({{unit}}_timeUnit)"
+}
```
Comment (anikdhabal) [inline apps/web/public/static/locales/tr/common.json:2104]: All text should be above this string :- `"ADD_NEW_STRINGS_ABOVE_THIS_LINE_TO_PREVENT_MERGE_CONFLICTS":`

**Comment by anikdhabal** [comment]
@zerone0x Also pls sign the CLA

File: apps/web/public/static/locales/tr/common.json (hunk 1 above)
Comment (anikdhabal) [reply apps/web/public/static/locales/tr/common.json:2104]: Let me fix that

Comment (zerone0x) [reply apps/web/public/static/locales/tr/common.json:2104]: How about no `ADD_NEW_STRINGS_ABOVE_THIS_LINE_TO_PREVENT_MERGE_CONFLICTS` key in json?

**Comment by zerone0x** [comment]
I did a quick fix, for those not included `ADD_NEW_STRINGS_ABOVE_THIS_LINE_TO_PREVENT_MERGE_CONFLICTS` key's json file, I just insert items into the bottom.

**Review by anikdhabal — DISMISSED** [review]
LGTM!!

**Review by Amit91848 — DISMISSED** [review]

**Comment by zerone0x** [comment]
@anikdhabal Could you help merge it?

**Comment by anikdhabal** [comment]
> @anikdhabal Could you help merge it?

Don't panic, we will merge it shortly

**Review by zomars — DISMISSED** [review]
Let's undo all unnecessarily changed files please 🙏

**Review by anikdhabal — APPROVED** [review]
Looks good now!!

**Comment by milospuac** [comment]
Can we revisit this? 
Screenshot 2024-08-21 at 12 24 45

**Comment by anikdhabal** [comment]
> Can we revisit this? Screenshot 2024-08-21 at 12 24 45

This is due to the recent changes in the duration component. However, the issue is that the new i18n string is not being automatically added to other locales apart from English

**Pull request merged**